k-distribution/src/main/scripts/bin/krun

```diff
@@ -564,7 +580,11 @@ if [ "$backendName" = "llvm" ]; then
     execute $cmdprefix "$llvmCommand" "$expanded_input_file" "${depth:--1}" "$disjunctFile" $interpreter_flags
     result=$?
     set -e
+    if [ -z "$noPattern" ]; then
     execute kore-match-disjunction "$kompiledDir/definition.kore" --module "$mainModuleName" --disjunction "$disjunctFile" --match "$patternFile" --output "$kore_output"
+    else
+    execute $cmdprefix "$kompiledDir/interpreter" "$expanded_input_file" "${depth:--1}" "$kore_output" $interpreter_flags
```

Comment:
This is not quite correct. What we want to do is to copy disjunctFile to kore_output rather than rerunning the interpreter.